wordpress-mobile/WordPress-iOS
# Removes the blogging reminders card.

Removes the blogging reminders card from the blog details VC.

## To test:

- Check that the layout of the main screen in the "My Site" tab looks good.
- The card was located in the blog details table view, right at the top.
- Load a site with Jetpack enabled and make sure the Jetpack title looks good as well.

## Regression Notes

1. Potential unintended areas of impact

My Site was affected by the change, but it should be enough to manually verify it's ok.  The removed code should be mostly fine as long as the App still builds.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:
